README.md

```diff
@@ -99,6 +99,30 @@ NAME      AGE
 example   15s
```

Comment:
Could u remove the earlier lines 
```
$ kubectl get zk
NAME      AGE
example   15s
```
as that would no longer be displayed?

Reply:
Thanks, I have removed these lines.